extension/lib/shopgate/CartExtensionPipeline.js

```diff
@@ -86,6 +86,14 @@ class ShopgateCartExtensionPipeline {
     return this._bigCommerceCartRepository.getCheckoutUrl()
   }
 
+  /**
+   * @param {Array} cartItemIds
```

Comment:
This should rather be an array of strings.

Reply:
done